notifier/selfstate/selfstate.go

```diff
@@ -55,13 +64,39 @@ func (selfCheck *SelfCheckWorker) Start() error {
 // Stop self check worker and wait for finish
 func (selfCheck *SelfCheckWorker) Stop() error {
```

Comment:
As I undestand It is bad system design  - each method must check `Enabled` flag. I will rewrite it to new design, in which SelfCheckWorker instance will be created (initialized) in main func only if `Enabled` is true

Reply:
no, i mean i didnt understand for what this "checks", Does it work without ?

Reply:
These checks (`if !Enabled`) need to prevent start of SelfCheckWorker in case when it is disabled in config. 
Seems it work without [these](https://github.com/moira-alert/moira/pull/807/files#diff-8fa47c332f569bcef75078b289ebacceb6e5f1c6c30ff92e323f47a1db8d5ad3R66-R72) lines. It is strange to check `Enabled` and check config in Stop method, I think checks in Start method is enough. Thanks 👍

Reply:
Fixed https://github.com/moira-alert/moira/pull/807/commits/ecea2cc7eb26929393d9105392077bf25949fbb6 and https://github.com/moira-alert/moira/pull/807/commits/5b51f3a9dc2f0d4ba47a5dac4744f6fad2311892